xds/src/test/java/io/grpc/xds/XdsLocalityTest.java

```diff
@@ -20,12 +20,13 @@
 
 import com.google.common.testing.EqualsTester;
 import io.envoyproxy.envoy.api.v2.core.Locality;
+import io.grpc.xds.ClusterLoadAssignmentData.XdsLocality;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 /**
- * Unit tests for {@link XdsLocality}.
+ * Unit tests for {@link ClusterLoadAssignmentData.XdsLocality}.
```

Comment:
May want to change the class name to `ClusterLoadAssignmentDataTest` and prepend `xdsLocality_` to all the existing methods and add similar test methods for other data types in `ClusterLoadAssignmentData`. Since this is trivial, maybe it's ok to not have tests for this, we can delete it totally. But now, it's half baked.

Reply:
Moved.